images/multus-cni/tests/deploy.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
```

Comment:
I don't see where we are testing our image here? where do we specify the image we're building and override it in the manifests?

Also is likely why this is passing whenever the package isn't included on the image?